python/lsst/meas/extensions/piff/piffPsfDeterminer.py

```diff
@@ -105,6 +105,10 @@ class PiffPsfDeterminerConfig(BasePsfDeterminerTask.ConfigClass):
         check=_validateGalsimInterpolant,
         default="Lanczos(11)",
     )
+    _debugStarData = pexConfig.Field[bool](
+        doc="Write used star image data to disk for debugging.",
```

Comment:
thought `PsfDeterminer` doesn't necessarily interact with a disk, so this is really more like "Include star images used for fitting in the PSF model/object"